Client/Frontend/Browser/TopTabCell.swift

```diff
@@ -0,0 +1,343 @@
+//
+//  TopTabCell.swift
+//  Client
+//
+//  Created by Tyler Lacroix on 6/27/16.
+//  Copyright © 2016 Mozilla. All rights reserved.
+//
+
```

Comment:
I renamed it and separated the top tabs layout into a new file, does that make more sense or should I split up the VC file further? :)